Documentation/running-tests.md

```diff
@@ -98,6 +98,21 @@ requires that the kernel be compiled with `CONFIG_BLK_DEV_ZONED` enabled.
 RUN_ZONED_TESTS=1
 ```
 
+### Running nvme-rdma nvmeof-mp srp tests
+
+Most of these tests will use the rdma_rxe (soft-RoCE) driver by default. The siw (soft-iWARP) driver is also supported.
+```sh
+To use the rdma_rxe driver:
+nvme-trtype=rdma ./check nvme/
+./check nvmeof-mp/
+./check srp/
+
+To use the siw driver:
+use_siw=1 nvme-trtype=rdma ./check nvme/
```

Comment:
I think that `nvme-trtype=rdma` can be left out since it is the default.

Reply:
the default was defined as loop: nvme_trtype=${nvme_trtype:-"loop"}
https://github.com/osandov/blktests/blob/master/tests/nvme/rc#L13